<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QMainWindow" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Window</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="GraphWindow">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>280</width>
      <height>280</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="send_pushButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>130</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Send_move</string>
    </property>
   </widget>
   <widget class="QComboBox" name="move_comboBox">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>90</y>
      <width>86</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="next_move_pushButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>350</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Next_move</string>
    </property>
   </widget>
   <widget class="QPushButton" name="prev_pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Prev_move</string>
    </property>
   </widget>
   <widget class="QPushButton" name="last_move_pushButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>350</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Last_move</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="color1_graphicsView">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="req_comboBox">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>240</y>
      <width>86</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="req_pushButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>290</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Request_game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="show_pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Show_requested_game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quit_pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resign_pushButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>420</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Resign</string>
    </property>
   </widget>
   <widget class="QPushButton" name="send_size_pushButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>230</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Send_Board-Size</string>
    </property>
   </widget>
   <widget class="QComboBox" name="size_comboBox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>150</y>
      <width>86</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="color2_graphicsView">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>You</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Oponent</string>
    </property>
   </widget>
   <widget class="QLabel" name="score_label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="Start_Game_p2_pushButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>180</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Start_Game</string>
    </property>
   </widget>
   <zorder>size_comboBox</zorder>
   <zorder>send_size_pushButton</zorder>
   <zorder>GraphWindow</zorder>
   <zorder>send_pushButton</zorder>
   <zorder>move_comboBox</zorder>
   <zorder>next_move_pushButton</zorder>
   <zorder>prev_pushButton</zorder>
   <zorder>last_move_pushButton</zorder>
   <zorder>color1_graphicsView</zorder>
   <zorder>req_comboBox</zorder>
   <zorder>req_pushButton</zorder>
   <zorder>show_pushButton</zorder>
   <zorder>quit_pushButton</zorder>
   <zorder>resign_pushButton</zorder>
   <zorder>color2_graphicsView</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>score_label</zorder>
   <zorder>Start_Game_p2_pushButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>555</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
